charts/victoria-metrics-k8s-stack/hack/sync_rules.py

```diff
@@ -24,172 +28,56 @@ def new_representer(dumper, data):
-    'kubernetes-apps': '.Values.defaultRules.rules.kubernetesApps',
-    'kubernetes-resources': '.Values.defaultRules.rules.kubernetesResources',
-    'kubernetes-storage': '.Values.defaultRules.rules.kubernetesStorage',
-    'kubernetes-system': '.Values.defaultRules.rules.kubernetesSystem',
-    'kubernetes-system-apiserver': '.Values.defaultRules.rules.kubernetesSystem',
-    'kubernetes-system-controller-manager': '.Values.kubeControllerManager.enabled',
-    'kubernetes-system-kubelet': '.Values.defaultRules.rules.kubernetesSystem',
-    'kubernetes-system-scheduler': '.Values.kubeScheduler.enabled .Values.defaultRules.rules.kubeScheduler',
-    'node-exporter': '.Values.defaultRules.rules.node',
-    'node-exporter.rules': '.Values.defaultRules.rules.node',
-    'node-network': '.Values.defaultRules.rules.network',
-    'node.rules': '.Values.defaultRules.rules.node',
-    'vmagent': '.Values.vmagent.enabled .Values.defaultRules.rules.vmagent',
-    'alertmanager.rules': '.Values.defaultRules.rules.alertmanager',
-    'vmcluster': '.Values.defaultRules.rules.vmcluster',
-    'vmsingle': '.Values.defaultRules.rules.vmsingle',
-    'vm-health': '.Values.defaultRules.rules.vmhealth'
-}
 
-alert_condition_map = {
-    'KubeAPIDown': '.Values.kubeApiServer.enabled',  # there are more alerts which are left enabled, because they'll never fire without metrics
-    'KubeControllerManagerDown': '.Values.kubeControllerManager.enabled',
-    'KubeSchedulerDown': '.Values.kubeScheduler.enabled',
-    'KubeStateMetricsDown': ' (index .Values "kube-state-metrics" "enabled")',  # there are more alerts which are left enabled, because they'll never fire without metrics
-    'KubeletDown': '.Values.kubelet.enabled',  # there are more alerts which are left enabled, because they'll never fire without metrics
-    'PrometheusOperatorDown': '.Values.prometheusOperator.enabled',
-    'NodeExporterDown': '.Values.nodeExporter.enabled',
-    'CoreDNSDown': '.Values.kubeDns.enabled',
-    'AlertmanagerDown': '.Values.alertmanager.enabled',
-    'KubeProxyDown': '.Values.kubeProxy.enabled',
-}
+def escape(s):
+    return s.replace("{{", "{{`{{").replace("}}", "}}`}}").replace("{{`{{", "{{`{{`}}").replace("}}`}}", "{{`}}`}}")
 
 replacement_map = {
     'https://runbooks.prometheus-operator.dev/runbooks': {
         'replacement': '{{ .Values.defaultRules.runbookUrl }}',
-        'init': ''},
+    },
     'job="kube-state-metrics"': {
-        'replacement': 'job="kube-state-metrics", namespace=~"{{ $targetNamespace }}"',
+        'replacement': 'job="kube-state-metrics", namespace=~"{{ .targetNamespace }}"',
         'limitGroup': ['kubernetes-apps'],
-        'init': '{{- $targetNamespace := .Values.defaultRules.appNamespacesTarget }}'},
+    },
     'job="kubelet"': {
-        'replacement': 'job="kubelet", namespace=~"{{ $targetNamespace }}"',
+        'replacement': 'job="kubelet", namespace=~"{{ .targetNamespace }}"',
```

Comment:
`.targetNamespace` can't be templated correctly.

Reply:
fixed template context